node/src/main/scala/coop/rchain/node/NodeRuntime.scala

```diff
@@ -134,7 +134,7 @@ class NodeRuntime private[node] (
       _   <- transport.shutdown(msg)
       _   <- log.info("Shutting down HTTP server....")
       _   <- Task.delay(Kamon.stopAllReporters())
-      _   <- LiftIO[Task].liftIO(servers.httpServer.shutdown).attempt
+      _   <- servers.httpServer.cancel
```

Comment:
I'm not sure if this is a proper way to shut down an http4s HTTP server

Reply:
In 0.19.0 they removed shutdown methods in favor to cats effects Resource and fs2 streams.
This code was basically taken from documentation to 0.19.0
https://http4s.org/v0.19/service/